securedrop/sass/source.sass

```diff
@@ -297,8 +297,24 @@ header
       padding-top: 0
       min-height: unset
 
-    p
+    &> section:not(#locales-menu):not(#flashed)
+
+      width: 100%
+      padding-top: 10px
+      min-height: unset
+
+      &:first-child
+        @include ltr
+          border-right: 0
+        @include rtl
+          border-left: 0
```

Comment:
if it's `width: auto`, will the sections be ordered vertically in all cases tho?

Reply:
Yes - for small screens we override the `main`'s `display: flex` with `display: block`, and since there's no float rules here at all, both `section`s should just show up one after the other!